src/notes/gitHub

```diff
@@ -1,6 +1,19 @@
-how to Update gitHub
```

Comment:
I think this file should be in the .gitignore 🤔

Reply:
ok